docs/user-guide.md

```diff
@@ -111,9 +134,9 @@ public class Strings {
 ```
 
 In this example, both methods are in the scope of `@NullMarked`, so plain `String`
-means "a reference to a `String` that can't be null". `@Nullable String` continues
-to mean "a reference to a `String` that can be null". Tools should warn you if you
-try to pass a "reference to a `String` that can be null" to `spaceIndex`, since its
+means "a string reference that can't be null". `@Nullable String` continues
```

Comment:
Here again, I really don't like condoning the idea that null IS a "string reference" at all.

Reply:
OK, will reword.